docs/IE_DG/supported_plugins/CL_DNN.md

```diff
@@ -115,7 +115,7 @@ When specifying key values as raw strings (that is, when using Python API), omit
 | `KEY_GPU_THROUGHPUT_STREAMS`  | `KEY_GPU_THROUGHPUT_AUTO`, or positive integer| 1 | Specifies a number of GPU "execution" streams for the throughput mode (upper bound for a number of inference requests that can be executed simultaneously).<br>This option is can be used to decrease GPU stall time by providing more effective load from several streams. Increasing the number of streams usually is more effective for smaller topologies or smaller input sizes. Note that your application should provide enough parallel slack (e.g. running many inference requests) to leverage full GPU bandwidth. Additional streams consume several times more GPU memory, so make sure the system has enough memory available to suit parallel stream execution. Multiple streams might also put additional load on CPU. If CPU load increases, it can be regulated by setting an appropriate `KEY_CLDNN_PLUGIN_THROTTLE` option value (see above). If your target system has relatively weak CPU, keep throttling low. <br>The default value is 1, which implies latency-oriented behavior.<br>`KEY_GPU_THROUGHPUT_AUTO` creates bare minimum of streams to improve the performance; this is the most portable option if you are not sure how many resources your target machine has (and what would be the optimal number of streams). <br> A positive integer value creates the requested number of streams. |
 | `KEY_EXCLUSIVE_ASYNC_REQUESTS` | `YES` / `NO`                | `NO`              | Forces async requests (also from different executable networks) to execute serially.|
-| `KEY_CLDNN_ENABLE_LOOP_UNROLLING` | `YES` / `NO`             | `YES`             | Enables recurrent layers such as TensorIterator or Loop with fixed iteration count to be unrolled. It is turned on by default. Note that unrolling loops will cause increase of graph loading time for GPU plugin in proportion to the iteration count, because the kernels in a body network are created and built for all iterations. On the other hand, disabling unrolling for loops with small iteration count and small body network might perform worse than unrolling, because the overhead for handling subnetworks might be more significant in overall performance.|
+| `KEY_CLDNN_ENABLE_LOOP_UNROLLING` | `YES` / `NO`             | `YES`             | Enables recurrent layers such as TensorIterator or Loop with fixed iteration count to be unrolled. It is turned on by default. Turning this key on will achieve better inference performance for loops with not too many iteration counts ( <= 16, as a rule of thumb). On the other hand, for loops with many iteration count (>= 16), turning this key off will achieve optimal performance for both graph loading time and inference. Note that loop unrolling will increase the graph loading time in proportion to the iteration counts. Thus, this key should be turned off if graph loading time is considered to be most important target to optimize. |
```

Comment:
In general, it looks good to me.

Some minor comments:

* I think it is not necessary to say `It is turned on by default`, as the table already says that it is on by default.
* It is difficult to understand the sentence below as the key condition is buried in the middle of sentence. It would be easier to read if you rewrite the sentence.
    * as-is: On the other hand, for loops with many iteration count (>= 16), turning this key off will achieve optimal performance for both graph loading time and inference.
    * suggestion: Turning this key off will achieve optimal performance for both graph loading time and inference with many iteration count (greater than 16)
* `optimal performance` -> `better performance`
* `Note that loop unrolling will...` --> `Note that turning key on will...`
* Is it intended whitespace between `(` and `<` in `( <= 16)`?